common/lib/container-definitions/api-report/container-definitions.api.md

```diff
@@ -132,8 +132,6 @@ export interface IContainer extends IEventProvider<IContainerEvents>, IFluidRout
     closeAndGetPendingLocalState(): string;
     readonly closed: boolean;
     connect(): void;
-    // @deprecated
-    readonly connected: boolean;
```

Comment:
@markfields, I believe you had some reservations about fully removing `IContainer.connected`/`IFluidContainer.connected`. Do you think we should reconsider this?

Reply:
Ok I've gone back and forth on this. but after looking at a few places in Loop code where this is used, I think you're right to get rid of it.  This is in light of the new state(s) I'm adding in #10416 - Each of the places where `connected` is used, requires some thought about how to interpret `TryingToConnect` and `Pending`.